packages/runtime/src/ctx.ts

```diff
@@ -70,6 +70,7 @@ export class Ctx implements Context {
 	remainingFee: number;
 	budget: number;
 	lastLog: Uint8Array;
+	txnType: TransactionType | undefined;
```

Comment:
lets add comment what it does and wy we need it

Reply:
Done